server.js

```diff
@@ -9,14 +11,57 @@ let fruits = [
   { id: "5", name: 'Orange', color: 'orange'}
 ]
 
+// @ GET - Display all
 app.get('/fruits', (req, res) => {
-  res.json(fruits)
+  const sortFruits = fruits.sort((a,b)=> a.id-b.id)
```

Comment:
Nice, finally I see someone using an array sorting function, did not happen for quite a while :) 

Just one remark:
Typically we never sort records by id, because in the frontend the Ids do not matter. We rather sort after name.

But I guess you just wanted to try it out, so that is actually good :)